ethereum/spec/src/test/java/tech/pegasys/teku/spec/datastructures/execution/ExpectedWithdrawalsTest.java

```diff
@@ -84,5 +86,55 @@ void electraExpectedWithdrawals() {
     assertThat(withdrawals.getWithdrawalList().get(1).getAmount())
         .isEqualTo(UInt64.valueOf(excessBalance));
     assertThat(withdrawals.getPartialWithdrawalCount()).isEqualTo(1);
+    final MutableBeaconStateElectra mutableBeaconStateElectra =
+        MutableBeaconStateElectra.required(preState.createWritableCopy());
+
+    withdrawals.processWithdrawalsUnchecked(
```

Comment:
It would be a nice addition to have an exited validator in the test state so we also exercise creating a full withdrawal in this test.

Reply:
im not sure its really in the same scope? does that use the partial withdrawal queue?

Reply:
It does not use the pending withdrawal queue, just the same flow when creating expected withdrawals. It will be exercised through reference tests anyway, so it is probably ok to ignore it.

Looking closer, this logic is already tested on `BlockProcessorCapellaTest#shoudlFindFullWithdrawals()` so we will be fine :)